### PR TITLE
move and rename some vars in `setParameters` and update the README

### PR DESCRIPTION
**Pull request by marcobarilari**

### CODE REVIEW

File: setParameters.m [hunk 1]
```diff
@@ -32,30 +32,38 @@
...
+    cfg.audio.channels = 2;
 
-    %% Task
+    %% Task(s)
 
-    cfg.task.name = 'visual localizer';
+    cfg.task.name = 'auditory localizer';
 
     % Instruction
     cfg.task.instruction = '1-Detect the RED fixation cross\n \n\n';
```
Comment (Remi-Gau) [inline setParameters.m:69]: Do we want to change the instructions to "detect longer sounds" or something like that?

File: subfun/expDesign.m [hunk 2]
```diff
@@ -48,11 +47,10 @@
 
     % Set variables here for a dummy test of this function
     if nargin < 1 || isempty(cfg)
-        cfg.names             = {'static', 'motion'};
-        cfg.numRepetitions    = 4;
-        cfg.speedEvent        = 4;
-        cfg.numEventsPerBlock = 12;
-        cfg.maxNumFixationTargetPerBlock = 2;
+        cfg.design.names             = {'static', 'motion'};
+        cfg.design.numRepetitions    = 4;
```
Comment (Remi-Gau) [inline subfun/expDesign.m:51]: For consistency I think I tried to switch all the `num` prefix to `nb` everywhere. 
For example, the lines below has `nb` and `Nb`.

But that migth require changes in other places then... Ah the joy of code cleaning... Sometimes if feels like a game of jenga tower.

File: setParameters.m (hunk 1 above)
Comment (marcobarilari) [reply setParameters.m:69]: yes, let me open an issue and will change this when I'll implement it

File: subfun/expDesign.m (hunk 2 above)
Comment (marcobarilari) [reply subfun/expDesign.m:52]: good catch, I missed that. fixed

File: setParameters.m [hunk 3]
```diff
@@ -35,9 +35,9 @@
     %% Experiment Design
 
     cfg.design.names              = {'static', 'motion'};
-    cfg.design.possibleDirections = [-1 1]; % 1 motion , -1 static
-    cfg.design.numBlocks          = size(cfg.possibleDirections, 2); % TO CHECK
-    cfg.design.numRepetitions     = 1;  % AT THE MOMENT IT IS NOT SET IN THE MAIN SCRIPT
+    cfg.design.possibleDirections = [-1 1]; % 1 motion , -1 static %NOT IN USE AT THE MOMENT
+    cfg.design.numBlocks          = size(cfg.design.names , 2); % TO CHECK
```
Comment (Remi-Gau) [inline setParameters.m]:
```suggestion
    cfg.design.nbBlocks          = size(cfg.design.names , 2); % TO CHECK
```

File: subfun/expDesign.m [hunk 4]
```diff
@@ -60,7 +60,7 @@
 
     % Get the parameters
     names = cfg.design.names;
-    numRepetitions = cfg.design.numRepetitions;
+    numRepetitions = cfg.design.nbRepetitions;
```
Comment (Remi-Gau) [inline subfun/expDesign.m:64]:
```suggestion
    nbRepetitions = cfg.design.nbRepetitions;
```

File: subfun/expDesign.m [hunk 5]
```diff
@@ -60,7 +60,7 @@
...
     numEventsPerBlock = cfg.design.nbEventsPerBlock;
```
Comment (Remi-Gau) [inline subfun/expDesign.m:65]:
```suggestion
    nbEventsPerBlock = cfg.design.nbEventsPerBlock;
```

File: subfun/expDesign.m [hunk 6]
```diff
@@ -48,11 +48,10 @@
 
     % Set variables here for a dummy test of this function
     if nargin < 1 || isempty(cfg)
-        cfg.names             = {'static', 'motion'};
-        cfg.numRepetitions    = 4;
-        cfg.speedEvent        = 4;
-        cfg.numEventsPerBlock = 12;
-        cfg.maxNumFixationTargetPerBlock = 2;
+        cfg.design.names             = {'static', 'motion'};
+        cfg.design.numRepetitions    = 4;
```
Comment (Remi-Gau) [inline subfun/expDesign.m]:
```suggestion
        cfg.design.nbRepetitions    = 4;
```

File: subfun/expDesign.m [hunk 7]
```diff
@@ -97,7 +95,6 @@
 
     cfg.designBlockNames      = cell(nrBlocks, 1);
     cfg.designDirections      = zeros(nrBlocks, numEventsPerBlock);
```
Comment (Remi-Gau) [inline subfun/expDesign.m]:
```suggestion
    cfg.designDirections      = zeros(nrBlocks, nbEventsPerBlock);
```

File: subfun/expDesign.m [hunk 8]
```diff
@@ -97,7 +95,6 @@
...
-    cfg.designSpeeds          = ones(nrBlocks, numEventsPerBlock) * speedEvent;
     cfg.designFixationTargets = zeros(nrBlocks, numEventsPerBlock);
```
Comment (Remi-Gau) [inline subfun/expDesign.m]:
```suggestion
    cfg.designFixationTargets = zeros(nrBlocks, nbEventsPerBlock);
```

File: setParameters.m [hunk 9]
```diff
@@ -32,30 +32,38 @@
...
-
     %% Experiment Design
-    cfg.names              = {'static', 'motion'};
-    cfg.possibleDirections = [-1 1]; % 1 motion , -1 static
-    cfg.numBlocks          = size(cfg.possibleDirections, 2);
-    cfg.numRepetitions     = 1;      % AT THE MOMENT IT IS NOT SET IN THE MAIN SCRIPT
+
+    cfg.design.names              = {'static', 'motion'};
+    cfg.design.possibleDirections = [-1 1]; % 1 motion , -1 static %NOT IN USE AT THE MOMENT
+    cfg.design.numBlocks          = size(cfg.design.names, 2); % TO CHECK
```
Comment (Remi-Gau) [inline setParameters.m:39]:
```suggestion
    cfg.design.nbBlocks          = size(cfg.design.names, 2); % TO CHECK
```

**Pull request merged**